tests/helpers/test_many.ts

```diff
@@ -61,21 +48,24 @@ export const makeTestMany = <M extends string>(
         }
 
         const variation = matrix[name]()
-        ;(variation as TestRunnerOptions).getCurrentBundlerName = getCurrentBundlerName.bind(null, variation.config)
```

Comment:
Instead of always binding this method for every test so it can be used as `options. getCurrentBundlerName()` I changed the API to be `getBundlerNameFromOptions(options)` instead. The function is not used that often, so binding a method for every test seems unnecessary.